kernel/arch/dreamcast/fs/fs_dcload.c

```diff
@@ -20,6 +20,7 @@ printf goes to the dc-tool console
 #include <dc/fs_dcload.h>
 #include <kos/thread.h>
 #include <arch/spinlock.h>
+#include <arch/arch.h>
```

Comment:
Needed for kos_init_flags. Previously it was accidentally chaining in from kos/thread